pkg/multus/multus.go

```diff
@@ -486,6 +488,59 @@ func cmdErr(k8sArgs *types.K8sArgs, format string, args ...interface{}) error {
 	return logging.Errorf(prefix+format, args...)
 }
 
+func convertResourceClaimToAnnotations(kubeClient *k8s.ClientInfo, pod *v1.Pod) error {
```

Comment:
or else, it could add a new file for the new code  in the module with entirely new license claim

Reply:
all add newly codes is at pkg/spiderpool